src/index.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+
+const state = {
+    currentTemp: 0,
+    clicked: false,
+    city: "Arcadia",
+    sky: "Sunny",
+};
+
+const increaseTemp = () => {
+    const tempContainer = document.getElementById("tempDisplay");
+    state.currentTemp +=1;
+    tempContainer.textContent = state.currentTemp;
+    tempColor();
+    addLandscape();
+    
+};
+
+const descreaseTemp = () => {
+    const tempContainer = document.getElementById("tempDisplay");
+    state.currentTemp -=1;
+    tempContainer.textContent = state.currentTemp;
+    tempColor();
+    addLandscape();
+};
+
+const tempColor =() => {
+    if (state.currentTemp <= 49){
+        tempDisplay.style.backgroundColor = "teal";
+    }
+    else if ((state.currentTemp > 49) && (state.currentTemp <= 59)){
+        tempDisplay.style.backgroundColor = "green";
+    }
+    else if ((state.currentTemp > 59) && (state.currentTemp <= 69)){
+        tempDisplay.style.backgroundColor = "yellow";
+    }
+    else if ((state.currentTemp > 69) && (state.currentTemp <= 79)){
+        tempDisplay.style.backgroundColor = "orange";
+    }
+    else if (state.currentTemp >= 80){
+        tempDisplay.style.backgroundColor = "red";
+    }
+};
+
+const addLandscape = () => {
+    const newLandscape = document.createElement("span");
+    const landscapeContainer = document.querySelector("#landscapeContainer");
+    let landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    if (state.currentTemp <= 59){
+        landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
+    else if ((state.currentTemp > 59) && (state.currentTemp <= 69)){
+        landscape = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    }
+    else if ((state.currentTemp > 69) && (state.currentTemp <= 79)){
+        landscape = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    }
+    else if (state.currentTemp >= 80){
+        landscape = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    }
+    landscapeContainer.textContent = landscape;
+}
+const getCityName = (event) => {
+    if (event.keyCode === 13){
+        document.getElementById('textbox_id').value = "";
+        return;
+    }
+    const cityNameInput = document.getElementById('textbox_id').value;
+    const cityNameHeader = document.getElementById("cityName");
+    cityNameHeader.textContent = cityNameInput;
+    state.city = cityNameInput;
+
+}
+const resetCity = () => {
+    state.city = "Arcadia";
+    const cityName = document.getElementById("cityName");
+    cityName.textContent = state.city;
+
+}
+const findLatitudAndLongitude = (query) =>{
+    let latitude, longitude;
+    axios.get('http://127.0.0.1:5000/location',{
+        params: {
+            q: state.city,
+        },
+    })
+    .then((response) => {
+        latitude = response.data[0].lat;
+        longitude = response.data[0].lon;
+        let currentTempWeather = findWeather(latitude,longitude);
```

Comment:
you aren't using this variable name anywhere in your function so you could just say
```suggestion
        findWeather(latitude,longitude);
```